numpy/core/_add_newdocs.py

```diff
@@ -2020,9 +2019,10 @@ def luf(lamdaexpr, *args, **kwargs):
 
     First mode, `buffer` is None:
 
+    >>> np.random.seed(123)
     >>> np.ndarray(shape=(2,2), dtype=float, order='F')
-    array([[ -1.13698227e+002,   4.25087011e-303],
-           [  2.88528414e-306,   3.27025015e-309]])         #random
+    array([[0.0e+000, 0.0e+000],
```

Comment:
Seeding isn't going to help - this is uninitialized memory

Reply:
One other advantage of a custom doctester: can recognize a comment-like human-readable marker, `# random` or `# may vary`.

Reply:
Yeah, I switched back to that approach now that we use refguide. Technically NumPy had its own checker but I guess it is long dead now vs. the SciPy one I've ported over.